docs/README.md

```diff
@@ -2449,6 +2449,100 @@ The following is the list of the fields you can configure:
   config.assets = ['.env']
   ```
 
+#### Providers configuration
+
+##### AWS Provider
+
+To configure AWS as a provider you need to meet certain prerequisites:
+
+- Set up an AWS account following the getting started section [instructions](#set-up-an-aws-account)
```

Comment:
Maybe it's worth moving that part of the doc in here? Since the `getting started` section is meant to be for all the cases? What do you think?